src/io/flutter/run/bazelTest/BazelTestFields.java

```diff
@@ -0,0 +1,252 @@
+    this(template);
+    if (StringUtil.isEmptyOrSpaces(launchScript)) {
+      launchScript = w.getLaunchScript();
+      if (launchScript != null && !launchScript.startsWith("/")) {
+        launchScript = w.getRoot().getPath() + "/" + launchScript;
+      }
+    }
+  }
+
+  /**
+   * The file containing the main function that starts the Flutter app.
+   */
+  @Nullable
+  public String getEntryFile() {
+    return entryFile;
+  }
+
+  public void setEntryFile(final @Nullable String entryFile) {
+    this.entryFile = entryFile;
+  }
+
+  /**
+   * Present only for deserializing old run configs.
+   */
+  @SuppressWarnings("SameReturnValue")
+  @Deprecated
+  public String getWorkingDirectory() {
+    return null;
+  }
+
+  /**
+   * Used only for deserializing old run configs.
+   */
+  @Deprecated
+  public void setWorkingDirectory(final @Nullable String workDir) {
+    if (entryFile == null && workDir != null) {
+      entryFile = workDir + "/lib/main.dart";
+    }
+  }
+
+  @Nullable
+  public String getLaunchingScript() {
+    return launchScript;
+  }
+
+  public void setLaunchingScript(@Nullable String launchScript) {
+    this.launchScript = launchScript;
+  }
+
+  //@Nullable
+  //public String getAdditionalArgs() {
+  //  return additionalArgs;
+  //}
+  //
+  //public void setAdditionalArgs(@Nullable String additionalArgs) {
+  //  this.additionalArgs = additionalArgs;
+  //}
+
+  @Nullable
+  public String getBazelTarget() {
+    return bazelTarget;
+  }
+
+  public void setBazelTarget(@Nullable String bazelTarget) {
+    this.bazelTarget = bazelTarget;
+  }
+
+  BazelTestFields copy() {
+    return new BazelTestFields(this);
+  }
+
+  @NotNull
+  BazelTestFields copyTemplateToNonTemplate(@NotNull final Project project) {
+    final Workspace workspace = WorkspaceCache.getInstance(project).getNow();
+    if (workspace == null) return new BazelTestFields(this);
+    return new BazelTestFields(this, workspace);
+  }
+
+  /**
+   * Reports an error in the run config that the user should correct.
+   * <p>
+   * This will be called while the user is typing into a non-template run config.
+   * (See RunConfiguration.checkConfiguration.)
+   *
+   * @throws RuntimeConfigurationError for an error that that the user must correct before running.
+   */
+  void checkRunnable(final @NotNull Project project) throws RuntimeConfigurationError {
+    // The UI only shows one error message at a time.
+    // The order we do the checks here determines priority.
+
+    final DartSdk sdk = DartPlugin.getDartSdk(project);
+    if (sdk == null) {
+      throw new RuntimeConfigurationError(FlutterBundle.message("dart.sdk.is.not.configured"),
+                                          () -> DartConfigurable.openDartSettings(project));
+    }
+
+    //final MainFile.Result main = MainFile.verify(entryFile, project);
+    //if (!main.canLaunch()) {
+    //  throw new RuntimeConfigurationError(main.getError());
+    //}
+
+    // check launcher script
```

Comment:
Nit: comments elsewhere start with a capital. 

=> // Check ...

Reply:
Fixed here and the other locations in this file.  I'll fix the occurrences in BazelFields.java in a follow up PR